keps/sig-apps/2879-ready-pods-job-status/README.md

```diff
@@ -221,20 +224,20 @@ The field is only informative, it doesn't affect running workloads.
 ###### What specific metrics should inform a rollback?
 
 - An increase in `job_sync_duration_seconds`.
-- A reduction in `job_sync_num`.
+- A reduction in `job_syncs_total`.
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-A manual test will be performed, as follows:
+A manual test on Beta was performed, as follows:
```

Comment:
This testing scenario is updated. One slight change compared to the scenario was point `Verify that ready field in Job A is not lost, but also not updated.`. In fact the Job's ready field is cleaned up. This is because after restart of the downgraded Job controller, all jobs are synced, due to the ADDED event.